src/action/auth-mobile.js

```diff
@@ -258,6 +270,41 @@ class AuthAction {
     const bytes = await this._Random.getRandomBytesAsync(32);
     return Buffer.from(bytes.buffer).toString('hex');
   }
+
+  //
+  // Help / Restore actions
+  //
+
+  askForHelp() {
+    const message =
+      "If you have forgotten your PIN, or you're locked out of your wallet, you can reset your PIN with your Recovery Phrase.";
+    const action = 'Recover My Wallet';
+    const cancel = 'Cancel';
+    if (this._Platform.OS === 'ios') {
+      this._ActionSheetIOS.showActionSheetWithOptions(
+        {
+          message,
+          options: [cancel, action],
+          cancelButtonIndex: 0,
+          destructiveButtonIndex: 1,
+        },
+        i => i === 1 && this._initRestoreWallet()
+      );
+    } else {
+      this._Alert.alert(null, message, [
+        {
+          text: action,
+          onPress: () => this._initRestoreWallet(),
+        },
+        { text: cancel, style: 'cancel' },
+      ]);
+    }
+  }
+
+  _initRestoreWallet() {
+    this._store.settings.restoring = true;
```

Comment:
minor: could move this line 305 into `wallet.initRestoreWallet`, then we can remove this helper

Reply:
Thought about that too. But seems a bit unclean since we set that before `wallet.initRestoreWallet()` in the select-seed screen.